packages/playground/blueprints/src/lib/steps/define-virtual-wp-config-consts.ts

```diff
@@ -0,0 +1,47 @@
+import { StepHandler } from '.';
+
+/**
+ * The step object for defining constants in the VFS_CONFIG_FILE_PATH php file and loaded using the auto_prepend_file php.ini directive.
+ */
+export interface DefineVirtualWpConfigConstsStep {
+	step: 'defineVirtualWpConfigConsts';
+	/** The constants to define */
+	consts: Record<string, unknown>;
+}
+
+const VFS_CONFIG_PATH = '/vfs-blueprints';
+const VFS_CONFIG_FILE_NAME = 'wp-config-consts.php';
+export const VFS_CONFIG_FILE_PATH = `${VFS_CONFIG_PATH}/${VFS_CONFIG_FILE_NAME}`;
+
+/**
+ * Function to build the contents of the config php file.
+ *
+ * @param {Record<string, unknown>} consts - An object containing constants to be defined.
+ * @returns {string} - Returns the file content as a string.
+ */
+function buildConfigFileContents(consts: Record<string, unknown>) {
+	let contents = `<?php `;
+	for (const [key, value] of Object.entries(consts)) {
+		contents += `if (!defined('${key}')) {
+      define("${key}", ${JSON.stringify(value)});
+    }`;
+	}
+	return contents;
+}
+
+/**
+ * Function to define constants in the virtual VFS_CONFIG_FILE_PATH php file of a WordPress installation.
+ * The file is then dinamically loaded using the auto_prepend_file php.ini directive.
+ *
+ * @param {Object} playground - The playground client.
+ * @param {DefineVirtualWpConfigConstsStep} wpConfigConst - An object containing the constants to be defined and the optional virtual file system configuration file path.
```

Comment:
Inline types aren't needed in TypeScript

Reply:
Right!, removed https://github.com/WordPress/wordpress-playground/pull/343/commits/7191df9dfcb9c4e4b5d81a0a7b9573374fe54178